### PR TITLE
Issue 172 task 2

### PR DESCRIPTION
**Pull request by a-ajith**
The PR contains the IsAuthorized method change in the lib-utilities

### CODE REVIEW

**Review by amar-shalgar — APPROVED** [review]
LGTM

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

**Review by bharath-b-hpe — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]
LGTM

File: lib-utilities/services/auth.go [hunk 1]
```diff
@@ -42,10 +43,15 @@ func IsAuthorized(sessionToken string, privileges, oemPrivileges []string) (int3
 		},
 	)
 	if err != nil && response == nil {
-		log.Printf("error: something went wrong with rpc call: %v", err)
-		return http.StatusInternalServerError, errResponse.InternalError
+		errMsg := fmt.Sprintf("error: something went wrong with rpc call: %v", err)
```
Comment (irappa-pattar) [inline lib-utilities/services/auth.go]: use `rpc call failed` instead of `something went wrong with rpc call`

**Review by jeevan-kamkar — APPROVED** [review]

**Review by irappa-pattar — APPROVED** [review]

**Review by irappa-pattar — CHANGES_REQUESTED** [review]
address the comment

File: lib-utilities/services/auth.go (hunk 1 above)
Comment (a-ajith) [reply lib-utilities/services/auth.go]: Done

**Review by irappa-pattar — APPROVED** [review]

**Review by skhan-anjum — APPROVED** [review]

**Review by Bharath-KKB — APPROVED** [review]

**Pull request merged**